docs/guide/custom_policy.rst

```diff
@@ -60,6 +60,16 @@ Custom Network Architecture
 One way of customising the policy network architecture is to pass arguments when creating the model,
 using ``policy_kwargs`` parameter:
 
+.. note::
+
+    An action net and a value net will be added **after** the layers specified in ``net_arch``, in order to have the right output dimensions and activation functions (e.g. Softmax for discrete actions).
+    
+    E.g. if you specify ``net_arch = [128, 64]``, ``activation_fn=nnReLU`` and you have an on-policy algorithm and a discrete action space with 10 actions:
+    
+    - the actor will have an architecture of ``[128, 64, 10]`` with a ``nn.ReLU`` activation function in all layers but the last one, which will have a ``nn.Softmax``;
+    - the critic will have an architecture of ``[128, 64, 1]`` with a ``nn.ReLU`` activation function in all layers.
+
+
```

Comment:
I would add this note after the python code. It might be clearer to use the example code to illustrate your point, what do you think? Something like "In the previous example, as CartPole's action space is...".

Reply:
And completing it with a visual illustration in the same style as what follows in the doc can also help with understanding. Something that might look like

```
          obs
          <4>
   /               \
 <32>             <32>
  |                 |
 <32>             <32>
  |                 |
 <2>               <1>
action            value
```

Reply:
Sounds good!

Also it could be nice to complete the first comment in the example code briefly specifying that an action net and a value net will be added on top, something like this:
```python
# Custom actor (pi) and value function (vf) networks
# of two layers of size 32 each with Relu activation function
# Note: an action net and a value net will be added on top of the pi and the vf nets, respectively
policy_kwargs = dict(activation_fn=th.nn.ReLU,
                     net_arch=[dict(pi=[32, 32], vf=[32, 32])])
```
What do you think? Does it repeat to much what will be written after the example code?

Reply:
> Also it could be nice to complete the first comment in the example code

Agree. But in your comment I would rather write that it is a single linear layer that is added, not a net. I feel like it is less confusing that way.

Reply:
I agree!

> And completing it with a visual illustration in the same style as what follows in the doc can also help with understanding. Something that might look like
> 
> ```
>           obs
>           <4>
>    /               \
>  <32>             <32>
>   |                 |
>  <32>             <32>
>   |                 |
>  <2>               <1>
> action            value
> ```

Just a note on this, doesn't Cartpole imply an actor net's output dimension of 1? I think this wouldn't be the most clear case to show, since the value net would also add a layer of dimension 1

Reply:
> Just a note on this, doesn't Cartpole imply an actor net's output dimension of 1?

No, [CartPole's action space is Discrete(2)](https://www.gymlibrary.dev/environments/classic_control/cart_pole/#action-space)

Reply:
(The output of the actor is `[prob of action_0, prob_of_action_1]`.)

Reply:
> > Just a note on this, doesn't Cartpole imply an actor net's output dimension of 1?
> 
> No, [CartPole's action space is Discrete(2)](https://www.gymlibrary.dev/environments/classic_control/cart_pole/#action-space)

Ah yeah sorry, I read it quickly and I thought it had a single output neuron with sigmoid function to decide between left or right with 0 and 1.

Anyway, I improved the doc as discussed in these comments 💪